dlk/python/dlk/templates/src/func/conv2d.cpp

```diff
@@ -134,24 +131,22 @@ void conv_general(
       for(T_UINT ki = 0; ki < p.kernel_height; ki++)
       {
         T_INT row = (wi * p.stride_along_height) - p.padding + ki;
-        inside_row = (row >= 0 && row < (T_INT) p.input_height);
-
+	if (row < 0 || row >= p.input_height) { current_kernel_index += p.kernel_width * p.kernel_depth; continue; }
+				   
```

Comment:
Please remove this kind of white space changes.

Reply:
fixed